.github/workflows/integration-tests.yml

```diff
@@ -151,6 +151,12 @@ jobs:
           entrypoint: /bin/bash
           args: -c "(/opt/bitnami/openldap/bin/ldapwhoami -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony||sleep 5) && /opt/bitnami/openldap/bin/ldapadd -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony -f src/Symfony/Component/Ldap/Tests/Fixtures/data/fixtures.ldif && /opt/bitnami/openldap/bin/ldapdelete -H ldap://ldap:3389 -D cn=admin,dc=symfony,dc=com -w symfony cn=a,ou=users,dc=symfony,dc=com"
 
+      - name: Composer Validate Main composer.json
+        run: composer validate --strict --no-ansi --no-interaction --with-dependencies
+
+      - name: Composer Validate composer.json Files in src/ Directory
+        run: find src -regex ".*composer.json" | xargs -n1 composer validate --strict --no-ansi --no-interaction --with-dependencies
```

Comment:
This will validate all 163 `composer.json` files in the `src/` directory under strict mode (which ensures that warnings will return a non-zero exit code as well, just like errors already do by default).

In case you are wondering why warnings matter, aside from ensuring that we keep the code up to the most recent standards, there are also external projects that depend on this level of strictness. For example, the [Nix package manager](https://github.com/NixOS/nixpkgs/blob/8213089f8dc69fe36b1d4cb5a4d8eeb3618ec357/pkgs/build-support/php/hooks/composer-install-hook.sh#L110), which is used to package up software for development environments and deployment tooling.